docker-compose.yaml

```diff
@@ -14,3 +14,28 @@ services:
       - ./0x21e8:/0x21e8
       - ./x21e8:/x21e8
     command: sh -c "uvicorn x21e8.main:app --host 0.0.0.0 --port 8000 --timeout-keep-alive 240 --reload"
+  'planetmint':
+    image: ghcr.io/planetmint/planetmint-aio:latest-aio
```

Comment:
Please use `latest` instead of `latest-aio`. The latter was just a f**k up I did in the beginning and it is pretty old now, see https://github.com/planetmint/planetmint/pkgs/container/planetmint-aio